ray-operator/controllers/ray/common/pod.go

```diff
@@ -185,8 +185,19 @@ func DefaultWorkerPodTemplate(instance rayv1alpha1.RayCluster, workerSpec rayv1a
 			// For more details, please refer to: https://docs.ray.io/en/latest/ray-core/configure.html#tls-authentication.
 			Env:          deepCopyRayContainer.Env,
 			VolumeMounts: deepCopyRayContainer.VolumeMounts,
-			// If users specify ResourceQuota for the namespace, the init container need to specify resource explicitly.
-			Resources: deepCopyRayContainer.Resources,
+			// If users specify a ResourceQuota for the namespace, the init container needs to specify resources explicitly.
+			// GKE's Autopilot does not support GPU-using init containers, so we explicitly specify the resources for the
+			// init container instead of reusing the resources of the Ray container.
+			Resources: v1.ResourceRequirements{
+				Limits: v1.ResourceList{
+					v1.ResourceCPU:    resource.MustParse("200m"),
+					v1.ResourceMemory: resource.MustParse("256Mi"),
+				},
+				Requests: v1.ResourceList{
+					v1.ResourceCPU:    resource.MustParse("200m"),
+					v1.ResourceMemory: resource.MustParse("256Mi"),
```

Comment:
A comment here with some indication of why these numbers were chosen would be useful, perhaps a brief reminder of what the init container does

Reply:
Updated https://github.com/ray-project/kuberay/pull/1470/commits/9039f24866f504f7cdfd82be7f277933fdb68602